components/tools/OmeroWeb/test/unit/test_marshal.py

```diff
@@ -186,15 +188,25 @@ def test_ellipse_marshal(self, basic_ellipse):
         assert 1.0 == marshaled['radiusX']
         assert 0.5 == marshaled['radiusY']
 
-    def test_rgba(self, basic_ellipse):
+    def test_rgba(self, basic_ellipse, basic_line, basic_polyline):
```

Comment:
Probably a good idea to include some values < 0. This has really tripped us up in the past. These are instances where the red component is > 127 (`0x7f`). Can be easily achieved by using the `ctypes` module:

```
In [1]: import ctypes

In [2]: ctypes.c_int(0x800000ff).value
Out[2]: -2147483393
```

/cc @will-moore